readthedocs/v3/views.py

```diff
@@ -0,0 +1,40 @@
+import django_filters.rest_framework as filters
+from rest_framework.authentication import SessionAuthentication, TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.renderers import JSONRenderer
+from rest_framework.throttling import UserRateThrottle, AnonRateThrottle
+from rest_flex_fields import FlexFieldsModelViewSet
+
+from readthedocs.projects.models import Project
+from readthedocs.restapi.permissions import IsOwner
+
+from .filters import ProjectFilter
```

Comment:
This references something that doesn't exist.

Reply:
Ups... I forget to push this file.